app/controllers/trainees/outcome_dates_controller.rb

```diff
@@ -31,5 +31,9 @@ def trainee_params
           MultiDateForm::PARAM_CONVERSION.fetch(key, key)
         end
     end
+
+    def authorize_trainee
+      authorize(trainee)
```

Comment:
Should we also be checking the state of the trainee in this policy call too? i.e. users shouldn't be recording outcome dates for trainees who are not in the state `trn_received?`

Reply:
Changed this to the policy `recommend_for_qts?` which checks trainee is in the `trn_received` state